go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -380,6 +461,29 @@ func appEnablementRegistration(appInstanceId string, appName string, appVersion
 	return nil
 }
 
+/*
+func appEnablementAppSupportSubscribe(appInstanceId string) error {
```

Comment:
Why is this code commented?
The service should subscribe for App Termination Notifications and reply with a Termination Confirmation after cleanup (i.e. removing its service from App enablement).
Is there another uncommitted change with this update?

Reply:
NOTE: Same comment applies for other services.

Reply:
was commented but is valid code... was done because we are not sending a registration to termination for now (code commented out until we enable it)